install.sh

```diff
@@ -54,9 +54,6 @@
 #
 # __________ Process command-line arguments and environment variables _____________
 
-# Make sure we really exit with correct status when piping function output
-set -o pipefail
-
```

Comment:
I think if you request the merge via a github pull request, then if the "files changed view" shows something that looks wrong, a manual `git merge --no-ff --no-commit master` should git you to a place where you can either resolve existing conflicts, or if errors are found upon inspecting the current state of the code, you can make changes and `git add` then `git commit # this should complete the merge commit... message should be pre-populated with Merge master into caffeinated...`